HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTracksterLinks_cfi.py

```diff
@@ -6,19 +6,52 @@
     layer_clustersTime = cms.InputTag("hltHgcalMergeLayerClusters","timeLayerCluster"),
     inferenceAlgo = cms.string('TracksterInferenceByDNN'),
     linkingPSet = cms.PSet(
-        algo_verbosity = cms.int32(0),
-        cylinder_radius_sqr = cms.vdouble(9, 9),
-        dot_prod_th = cms.double(0.97),
-        max_distance_projective_sqr = cms.vdouble(60, 60),
-        max_distance_projective_sqr_closest_points = cms.vdouble(60, 60),
-        max_z_distance_closest_points = cms.vdouble(35, 35),
-        min_distance_z = cms.vdouble(30, 30),
-        min_num_lcs = cms.uint32(7),
-        min_trackster_energy = cms.double(10),
-        pca_quality_th = cms.double(0.85),
-        track_time_quality_threshold = cms.double(0.5),
-        type = cms.string('Skeletons'),
-        wind = cms.double(0.036)
+      cylinder_radius_sqr_split = cms.double(9),
```

Comment:
Hi Marco! Thanks for the comments. I fixed the code with your suggestions.
Regarding the `PSet`. In principle we would like to be able to have different selections depending on the seeding regions. We still need to study this, but in principle, we would like to have the `PSet`s for the different modules separate so that we can tune the parameters for each module.

Reply:
That's fine of course, but then you could still reference two different global `PSets` when that becomes necessary (no strong push for that,  just it looks a tiny bit less error prone)

Reply:
Hi @mmusich, before committing, I wanted to double-check with you if what I did makes sense.
I created a PSet `hltTiclTracksterLinksPSet_cfi.py` (not sure about the naming of the file, I couldn't spot a general convention from the other files) in `HLTrigger/Configuration/python/HLT_75e33/psets` 

I am then importing this PSet in `HLTrigger/Configuration/python/HLT_75e33/modules`

```diff
diff --git a/HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTracksterLinksL1Seeded_cfi.py b/HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTracksterLinksL1Seeded_cfi.py
index a5c9f73b188..d89b5ea6233 100644
--- a/HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTracksterLinksL1Seeded_cfi.py
+++ b/HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTracksterLinksL1Seeded_cfi.py
@@ -1,58 +1,12 @@
 import FWCore.ParameterSet.Config as cms
+from ..psets.hltTiclTracksterLinksPSet_cfi import hltTiclTracksterLinksPSet 
 
 hltTiclTracksterLinksL1Seeded = cms.EDProducer("TracksterLinksProducer",
     detector = cms.string('HGCAL'),
     layer_clusters = cms.InputTag("hltHgcalMergeLayerClustersL1Seeded"),
     layer_clustersTime = cms.InputTag("hltHgcalMergeLayerClustersL1Seeded","timeLayerCluster"),
     inferenceAlgo = cms.string('TracksterInferenceByDNN'),
-    linkingPSet = cms.PSet(
-      cylinder_radius_sqr_split = cms.double(9),
-      proj_distance_split = cms.double(5),
-      track_time_quality_threshold = cms.double(0.5),
-      min_num_lcs = cms.uint32(15),
-      min_trackster_energy = cms.double(20),
-      pca_quality_th = cms.double(0.85),
-      dot_prod_th = cms.double(0.97),
-      deltaRxy = cms.double(4),
-      lower_boundary = cms.vdouble(
-        20,
-        10
-      ),
-      upper_boundary = cms.vdouble(
-        150,
-        100
-      ),
-      upper_distance_projective_sqr = cms.vdouble(
-        30,
-        60
-      ),
-      lower_distance_projective_sqr = cms.vdouble(
-        30,
-        60
-      ),
-      min_distance_z = cms.vdouble(
-        35,
-        35
-      ),
-      upper_distance_projective_sqr_closest_points = cms.vdouble(
-        5,
-        30
-      ),
-      lower_distance_projective_sqr_closest_points = cms.vdouble(
-        10,
-        50
-      ),
-      max_z_distance_closest_points = cms.vdouble(
-        35,
-        35
-      ),
-      cylinder_radius_sqr = cms.vdouble(
-        9,
-        15
-      ),
-      algo_verbosity = cms.int32(0),
-      type = cms.string('Skeletons')
-    ),
+    linkingPSet = hltTiclTracksterLinksPSet,
     pluginInferenceAlgoTracksterInferenceByDNN = cms.PSet(
         algo_verbosity = cms.int32(0),
         onnxPIDModelPath = cms.FileInPath('RecoHGCal/TICL/data/ticlv5/onnx_models/linking/id_v0.onnx'),
```

And same thing for the L1Seeded version of the module. Does it look reasonable to you?

Reply:
>  Does it look reasonable to you?

yes, it does!